### ISSUE TITLE
consider archiving

### ISSUE THREAD

**Issue by tannerblair** [NIVeriStandAdd-Ons/Ethernet-IP-Custom-Device#1]

**Comment by AndrewHeim** [comment]
I agree that we could archive this.  It's worth pointing out that this is mostly because it hasn't been changed recently.  Perhaps we aren't directly exposed to its use.  It is considered older, well-tested, and so it is of good quality.
